### PR TITLE
Inject promtail manifests in SNO BIP workflow

### PR DESCRIPTION
**Pull request by vrutkovs**
This creates promtail manifests to be injected in the cluster and send manifest/service logs to Loki.

Required for https://github.com/openshift/assisted-test-infra/pull/1981

### CODE REVIEW

File: ci-operator/step-registry/baremetalds/sno/setup/baremetalds-sno-setup-commands.sh [hunk 1]
```diff
@@ -42,6 +42,13 @@ then
   scp "${SSHOPTS[@]}" "${SHARED_DIR}/sno-additional-config" "root@${IP}:sno-additional-config"
 fi
 
+# Copy additional manifests
+if [[ -e "${SHARED_DIR}/manifest_*" ]]
+then
+  ssh "${SSHOPTS[@]}" "root@${IP}" "mkdir sno-additional-manifests"
+  scp "${SSHOPTS[@]}" "${SHARED_DIR}/manifest_*" "root@${IP}:sno-additional-manifests"
```
Comment (eranco74) [inline ci-operator/step-registry/baremetalds/sno/setup/baremetalds-sno-setup-commands.sh:49]: `ipi-install-hosted-loki` will create manifest_... under ${SHARED_DIR}/ ?

Comment (vrutkovs) [reply ci-operator/step-registry/baremetalds/sno/setup/baremetalds-sno-setup-commands.sh:49]: Yup - https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/install/hosted-loki/ipi-install-hosted-loki-commands.sh#L29 and so on

**Comment by vrutkovs** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-metal-single-node-live-iso

**Comment by vrutkovs** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-metal-single-node-live-iso

**Comment by vrutkovs** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-metal-single-node-live-iso

**Comment by vrutkovs** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-metal-single-node-live-iso

**Comment by vrutkovs** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-metal-single-node-live-iso

**Pull request merged**